msprime/ancestry.py

```diff
@@ -1146,6 +1146,128 @@ def sim_ancestry(
     )
 
 
+def continue_simulation(
+    ts,
+    time,
+    sample_size=None,
+    continue_nodes=None,
+    **kwargs,
+):
+    """
+    Returns a full tree sequence, by continuing the simulation using msprime
+    after an earlier simulation has been performed. The msprime simulation and slim
+    simulation are joined using :meth:`tskit.tables.TableCollection.union()`.
+
+    This method executes code from the pyslim tutorial  'Following up with more
+    coalescent simulation'
+
+    ``sample_size`` defaults to the number of samples present in the input tree sequence.
```

Comment:
Documentation needs some work (e.g., remove reference to SLiM, since this is generic).